Index: configure.in
===================================================================
RCS file: /cvsroot/gcc/gcc/configure.in,v
retrieving revision 1.354
diff -u -r1.354 configure.in
--- configure.in	7 Jun 2005 09:32:14 -0000	1.354
+++ configure.in	7 Jun 2005 18:28:54 -0000
@@ -482,7 +482,7 @@
     noconfigdirs="$noconfigdirs target-libstdc++-v3 target-libgloss ${libgcj}"
     ;;
   c54x*-*-* | tic54x-*-*)
-    noconfigdirs="$noconfigdirs target-libstdc++-v3 target-libgloss ${libgcj} gcc gdb newlib"
+    noconfigdirs="$noconfigdirs target-libstdc++-v3 target-libgloss ${libgcj} gdb newlib"
     ;;
   cris-*-* | crisv32-*-*)
     unsupported_languages="$unsupported_languages java"
Index: gcc/config.gcc
===================================================================
RCS file: /cvsroot/gcc/gcc/gcc/config.gcc,v
retrieving revision 1.548
diff -u -r1.548 config.gcc
--- gcc/config.gcc	6 Jun 2005 09:46:35 -0000	1.548
+++ gcc/config.gcc	7 Jun 2005 18:28:54 -0000
@@ -322,6 +322,9 @@
 tic4x-*-*)
         cpu_type=c4x
         ;;
+tic54x-*-*)
+        cpu_type=c54x
+        ;;
 esac
 
 tm_file=${cpu_type}/${cpu_type}.h
@@ -762,6 +765,8 @@
 	c_target_objs="c4x-c.o"
 	cxx_target_objs="c4x-c.o"
 	;;
+c54x-* | tic54x-*)
+	;;
 cris-*-aout)
 	tm_file="dbxelf.h ${tm_file} cris/aout.h"
 	gas=yes
